legal-api/report-templates/correction.html

```diff
@@ -51,5 +51,10 @@
          [[correction/rulesMemorandum.html]]
          [[correction/resolution.html]]
       </div>
+      <div class="container business-details-table" >
+         <div class="bold"><span class="details-header-text"> Correction Detail</span><span class="correction-label-added">ADDED</span></div>
+         <div class="pt-2">Correction filed by {{correction.type}} on {{effective_date_time}}</div>
+         <div class="pt-2"><span class="bold">{{correction.comment.split('\n')[0]}}. </span><span>{{correction.comment.split('\n')[1]}}</span></div>
```

Comment:
Its better to not access lines with index. Try this (add `<br>` if needed)
```suggestion
         <div class="pt-2">
         <span class="bold">
          {% for line in correction.comment.split('\n') %}
              {{ line }}
          {% endfor %}
          </span>
          </div>
```